config/gulp/sass.js

```diff
@@ -10,6 +11,13 @@ var task = /([\w\d-_]+)\.js$/.exec(__filename)[ 1 ];
 
 var entryFileFilter = filter('all.scss', { restore: true });
 var normalizeCssFilter = filter('normalize.css', { restore: true });
+var supportedBrowsers = [
+  '> 1%',
+  'Last 2 versions',
+  'IE 11',
+  'IE 10',
+  'IE 9',
+];
```

Comment:
This current browser support [is mentioned here](https://standards.usa.gov/getting-started/) starting after `Browser support:`.